File: GTTS/INL.MessagingService/Models/NotificationVettingBatchRejected_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.MessagingService.Models
{
	public class NotificationVettingBatchRejected_Item : INotificationVettingBatchRejected_Item
	{
		public long VettingBatchID { get; set; }
		public int VettingBatchTypeID { get; set; }
		public string VettingBatchType { get; set; }		

		public string Name { get; set; }
		public int OrganizerAppUserID { get; set; }
		public int AppUserIDSubmitted { get; set; }
		public string BatchRejectionReason { get; set; }

		public DateTime EventStartDate { get; set; }
		public DateTime EventEndDate { get; set; }

		public int ParticipantsCount { get; set; }

		public string BatchViewURL { get; set; }

		public List<dynamic> Stakeholders { get; set; }

		public string EventStart
		{
			get
			{
				return String.Format("{0:MM/dd/yyyy}", this.EventStartDate);
			}
		}
		public string EventEnd
		{
			get
			{
				return String.Format("{0:MM/dd/yyyy}", this.EventEndDate);
			}
		}
	}
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/notification-vetting-batch-rejected_item.ts



export class NotificationVettingBatchRejected_Item  {
  
	public VettingBatchID: number = 0;
	public VettingBatchTypeID: number = 0;
	public VettingBatchType: string = "";
	public Name: string = "";
	public OrganizerAppUserID: number = 0;
	public AppUserIDSubmitted: number = 0;
	public BatchRejectionReason: string = "";
	public EventStartDate: Date = new Date(0);
	public EventEndDate: Date = new Date(0);
	public ParticipantsCount: number = 0;
	public BatchViewURL: string = "";
	public Stakeholders?: any[];
	public EventStart: string = "";
	public EventEnd: string = "";
  
}


